helpdesk_project/tests/test_helpdesk_project.py

```diff
@@ -0,0 +1,54 @@
+#Ticket in a project
+'''
+from odoo.tests import common
+
+@classmethod
+class TestHelpdeskTicketProject(common)
+    def ticket_in_proyect(cls):
+        super(TestHelpdeskTicketProject, cls).setUpClass()
+        
+        Ticket = env['helpdesk.ticket']
+        Project = env["project.project"]
+        
+        #ticket belongs to a project
+        -we create the new ticket
+        cls.ticket = Ticket.create({
+            'name': 'Test',
+            'description': 'Ticket test',
+        })
+        
+        -assign the new ticket to a project
+        
+        cls.project = Project.create({
+            'name': 'Test Helpdesk-Project',
+        })
+        cls.ticket.write({
+            'project_id': cls.project.id,
+        })
+        
+        -button ticket count +1
+        cls.project.write({
+            'ticket_count': cls-ticket_count+1
+        })
+        -check the count of the button is right
+    
+    def test_helpdesk_ticket_project(self):
+        self.assertNotEquals(self.ticket.project, '/',
+                             'Helpdesk Ticket: A ticket should have '
+                             'a project.')
+                             
+    def test_helpdesk_ticket_counts(self):
+        self.assertequal(self.project_id.ticket_count,
+                         1,
+                         'Helpdesk Ticket: Project assigned '
+                         'one ticket')
+
+        self.helpdesk_ticket.write({
+            'stage_id': self.stage_closed.id,
+        })
+
+        self.assertequal(self.project_id.ticket_count,
```

Comment:
makes sense this test?, You set a ticket in project, and check the project ticket count, then, change de ticket state, and again check the project ticket count... Think that you are not checking the number of open tickets, if not the total. If you were checking the open ones the cash should go down one, also the field name should be representative of the state (open_ticket_count for example)